app/javascript/quantity.js

```diff
@@ -0,0 +1,47 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const incrementButtons = document.querySelectorAll('[data-action="increment"]');
+  const decrementButtons = document.querySelectorAll('[data-action="decrement"]');
+  const quantityInputs = document.querySelectorAll('.quantity-input');
+  const priceTags = document.querySelectorAll('.price-tag');
+
+  incrementButtons.forEach((button, index) => {
+    button.addEventListener('click', () => {
+      let currentValue = parseInt(quantityInputs[index].value);
+      quantityInputs[index].value = currentValue + 1;
+      updateProductPrice(index, quantityInputs, priceTags);
+    });
+  });
+
+  decrementButtons.forEach((button, index) => {
+    button.addEventListener('click', () => {
```

Comment:
make this increment and decrement button into one method? where on clicking the + button the value is increased and on clicking the minus the value is decreased